03b_model/src/run_model.py

```diff
@@ -54,12 +99,41 @@ def select_inputs_targets(inputs, target, train_start_date, test_end_date, out_d
     #inputs_df = inputs_df.join(nerrs_input)
     inputs_df = inputs_df[train_start_date:test_end_date]
     
+    if inc_ante:
+        
+        with open("03b_model/model_config.yaml", 'r') as stream:
+            config = yaml.safe_load(stream)
+            
+        win_sizes = config['window_size']
+        ante_var = config['antecedant_variables']
+        
+        
+        if 'discharge' in ante_var:
+            inputs_df['discharge'] = inputs_df['discharge_01463500']+inputs_df['discharge_01474500']
```

Comment:
Right now these are the only two gages that we're using 01463500 = Delaware River at Trenton, 01474500 = Schuylkill River at Philadelphia. In the future, we might have more gages and it might make sense to sum over that list